examples/stable_diffusion.py

```diff
@@ -621,20 +622,23 @@ def get_x_prev_and_pred_x0(x, e_t, index):
     x_prev = math.sqrt(a_prev) * pred_x0 + dir_xt #+ noise
     return x_prev, pred_x0
 
+  @TinyJit
+  def do_step(latent, timestep):
+    e_t = get_model_output(latent, timestep)
+    x_prev, pred_x0 = get_x_prev_and_pred_x0(latent, e_t, index)
+    #e_t_next = get_model_output(x_prev)
+    #e_t_prime = (e_t + e_t_next) / 2
+    #x_prev, pred_x0 = get_x_prev_and_pred_x0(latent, e_t_prime, index)
+    return x_prev.realize(), pred_x0
```

Comment:
`pred_x0` needs a realize, not sure how this works without it

Reply:
it was not used, cleaned this a bit.